README.md

```diff
@@ -74,12 +74,16 @@ This will take a while.  We intend to provide an example in the wiki and/or samp
 
 ## EB Rake Tasks
 
-    rake eb:clobber                 # Remove any generated package
-    rake eb:config                  # Setup AWS.config and merge/override environments into one resolved configuration
-    rake eb:deploy[version]         # Deploy to Elastic Beanstalk
-    rake eb:destroy[force]          # ** Warning: Destroy Elastic Beanstalk application and *all* environments
-    rake eb:package                 # Package zip source bundle for Elastic Beanstalk
-    rake eb:show_config[version]    # Show resolved configuration without doing anything
+    rake eb:clobber                                       # remove any generated package
+    rake eb:config[environment,version]                   # setup aws.config and merge/override environments into one resolved configuration
+    rake eb:deploy[environment,version]                   # deploy to elastic beanstalk
+    rake eb:destroy[force]                                # ** warning: destroy elastic beanstalk application and *all* environments
+    rake eb:package[environment,version]                  # package zip source bundle for elastic beanstalk and generate external rakefile
+    rake eb:rds:create_snapshot[instance_id,snapshot_id]  # creates an rds snapshot
+    rake eb:rds:instances                                 # list rds instances
+    rake eb:rds:snapshots                                 # list rds snapshots
```

Comment:
These are listed below in a separate header, don't need to be included in this section